benches/benches/block_target_gas.rs

```diff
@@ -29,6 +39,17 @@ use fuel_core_types::{
     fuel_types::AssetId,
 };
 
+#[path = "utils.rs"]
```

Comment:
It should work without `path`

Reply:
Now that I've moved `block_target_gas` into its own module thing, it won't. But it's true in the old structure that I could have.

```
error[E0583]: file not found for module `utils`
  --> benches/benches/block_target_gas/main.rs:43:1
   |
43 | mod utils;
   | ^^^^^^^^^^
   |
   = help: to create the module `utils`, create file "benches/benches/block_target_gas/utils.rs" or "benches/benches/block_target_gas/utils/mod.rs"
```